testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -40,6 +40,8 @@
 
     SUPPORTS_CREATE_TABLE,
     SUPPORTS_CREATE_TABLE_WITH_DATA(SUPPORTS_CREATE_TABLE),
+    SUPPORTS_RENAME_TABLE(SUPPORTS_CREATE_TABLE),
```

Comment:
I don't think it should fallback to `SUPPORTS_CREATE_TABLE`. Rename is quite independent. Notice that on the other hand `RENAME` is not a specific case of `CREATE`. I can imagine connector with `RENAME` but without `CREATE`

Reply:
Done!

Reply:
If a connector doesn't support CREATE, it can be reasonably expected not to support RENAME
This is just to reduce amount of declarations in subclasses, since the truthfulness of the declaration is validated.

anyway, whatever